utils/copy-env-vars/copy-env-vars.ts

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+
+const folders = [
+  'dtfs-central-api',
+  'external-api',
+  'gef-ui',
+  'portal',
+  'portal-api',
+  'trade-finance-manager-api',
+  'trade-finance-manager-ui',
+  'utils',
+  path.join('azure-functions', 'acbs-function'),
+  path.join('libs', 'common'),
+];
+
+const basePath = path.join(__dirname, '..', '..');
+const baseEnvFilePath = path.join(basePath, '.env');
+
+/**
+ * Due to our current setup, the .env on the root on the project
+ * requires a slightly different setup than the .env that are copied to
+ * the .env that are used in the docker files
+ *
+ * This function updates the .env file to apply these rules
+ * @param fileData
+ */
+const updateEnvVars = (fileData: string): string => {
+  const MONGODB_URI_REGEX = /(MONGODB_URI=.*)(dtfs-submissions-data)/g;
+  const MONGODB_URI_QA_REGEX = /(MONGODB_URI_QA=.*)(dtfs-submissions-data)/g;
+  const SQL_DB_HOST_REGEX = /(SQL_DB_HOST=.*)(dtfs-sql)/g;
+
+  const replacePattern = '$1localhost';
+
+  const regexPatternsToApply = [MONGODB_URI_REGEX, MONGODB_URI_QA_REGEX, SQL_DB_HOST_REGEX];
+
+  return regexPatternsToApply.reduce(
+    (updatedFileData, regexPattern) => updatedFileData.replace(regexPattern, replacePattern),
+    fileData,
+  );
+};
+
+const copyEnvFileToFolders = () => {
+  try {
+    const envFileData = fs.readFileSync(baseEnvFilePath, 'utf8');
+    const newEnvFileData = updateEnvVars(envFileData);
+
+    folders.forEach((folderPathToCopyTo: string) => {
+      try {
+        const envVarPathToCopyTo = path.join(basePath, folderPathToCopyTo, '.env');
+        fs.writeFileSync(envVarPathToCopyTo, newEnvFileData);
+        console.info(`.env file copied to ${folderPathToCopyTo}`);
+      } catch (error) {
+        console.error(`Error copying .env file to ${folderPathToCopyTo}:`, error);
+        throw error;
+      }
+    });
+  } catch (error) {
+    console.error('Error copying env vars:', error);
+  }
+};
+
+copyEnvFileToFolders();
+
+console.info('Done!');
```

Comment:
Minor pedantic comment can you please add ✅ to keep all successful utils completion consistent. Thanks